test/variational/test_continuous_expect.py

```diff
@@ -59,3 +59,24 @@ def test_expect():
     """
     np.testing.assert_allclose(0.1975164, sol.mean, atol=10 ** (-3))
     np.testing.assert_allclose(-0.140256, O_grad, atol=10 ** (-3))
+
+
+def test_expect_chunked():
+    vs_continuous.chunk_size = 100
+    vs_continuous2.chunk_size = 100
+
```

Comment:
Can you also add a test that `chunk_size=100` yields the same result as with `chunk_size=None` for the gradient and the expect?